frontend/src/components/CloseChannelDialogContent.tsx

```diff
@@ -107,6 +113,15 @@ export function CloseChannelDialogContent({ alias, channel }: Props) {
               Are you sure you want to close the channel with {alias}?
             </AlertDialogTitle>
             <AlertDialogDescription className="text-left">
+              <Alert className="mb-4">
+                <AlertCircleIcon className="h-4 w-4" />
+                <AlertDescription>
+                  Closing this channel will move{" "}
+                  {formatAmount(channel.localBalance)} sats in this channel to
+                  your on-chain balance and reduce your receive limit by{" "}
+                  {formatAmount(channel.remoteBalance)} sats.
```

Comment:
I'm not sure this is 100% correct (showing the remote balance as the receive limit) as you can pay from the channel and then you would have more receiving capacity. But I guess it's ok

Reply:
> as you can pay from the channel

wdym? the channel is/will be closed right? I think it's good to explain this, otherwise people might think why the receiving capacity has been reduced

Reply:
@im-adithya e.g. if you had a 1M channel but only 100K receiving capacity (~900K was on your side of the channel), 100K we will show as being "lost". But technically you lose the possibility of losing up to near 1M sats receiving capacity.

But I guess it's ok as-is